File: Php/app/Urls/CashgamePublishUrlModel.php
<?php
namespace app\Urls{

	use app\RouteFormats;
	use app\Urls\BaseClasses\CashgameUrlModel;
	use entities\Cashgame;
	use entities\Homegame;

	class CashgamePublishUrlModel extends CashgameUrlModel{

		public function __construct(Homegame $homegame, Cashgame $cashgame){
			parent::__construct(RouteFormats::cashgamePublish, $homegame, $cashgame);
		}

	}

}

File: Web/Models/UrlModels/CashgamePublishUrlModel.cs
using Core.Classes;
using Web.Routing;

namespace Web.Models.UrlModels{
    public class CashgamePublishUrlModel : CashgameUrlModel{

		public CashgamePublishUrlModel(Homegame homegame, Cashgame cashgame)
			: base(RouteFormats.CashgamePublish, homegame, cashgame)
        {
		}

	}

}